torch/_dynamo/eval_frame.py

```diff
@@ -608,6 +609,10 @@ def export(
         tracing_mode (str): Specifies the tracing mode. Must be set to "real" if decomposition_table is not specified.
         If decomposition_table is specified, the options are "symbolic" or "fake". Default is "real".
 
+        constraints List[Constraint]: A list of user specified constraints, see Note - [On Export Dynamic Dimension UX]
```

Comment:
You can't reference Notes from docblocks, they won't actually show up in the generated documentation. Please properly format, or, probably easier, just put the proper discussion here in the docblock.